src/data/pages/contribute/index.js

```diff
@@ -171,6 +172,90 @@ const data = {
+            company: 'Facebook',
+          },
+          {
+            name: 'Daniel Borkmann',
+            company: 'Cilium/Isovalent',
+          },
+        ],
+        links: [
+          {
+            text: 'Git trees',
+            to: 'https://git.kernel.org/?q=BPF+Group',
+          },
+          {
+            text: 'Linux kernel eBPF runtime',
+            to: '/infrastructure/#linux-kernel',
+          },
+        ],
+      },
+    },
+    howToContribute: {
+      items: [
+        {
+          icon: 'file',
+          title: 'Contribute to an eBPF project',
+          description:
+            '<p>The list of eBPF-based projects is long and growing. It will be simple to find a project that sparks interest. Check out the eBPF landscape to see an overview of eBPF-based projects.</p><p>Many of the listed eBPF-based projects maintain a list of good-first-issue-labeled tasks which are scoped to not require extensive project specific knowledge and provide a great opportunity to get hands-on quickly.</p>',
+          links: [
+            {
+              text: 'eBPF landscape',
+              to: '/applications',
+            },
+          ],
+        },
+        {
+          icon: 'gear',
+          title: 'Contribute to the eBPF development toolchains',
+          description:
+            '<p>Even though the Linux kernel provides minimal libraries and APIs to use eBPF directly, the majority of projects leverage an eBPF development toolchain to simplify use of eBPF and share common code among other projects.</p><p>Check out eBPF development toolchains for a frequently updated list of projects in this space. You will find information on how to get involved on each project website directly.</p>',
+          links: [
+            {
+              text: 'eBPF development toolchains',
+              to: '/what-is-ebpf/#development-toolchains',
+            },
+          ],
+        },
+        {
+          icon: 'pen',
+          title: 'Contribute to ebpf.io',
+          description:
+            '<p>Do you have a writing talent? Are you good with diagrams? The content of the ebpf.io website is licensed under a Creative Commons Attribution 4.0 International License. If you want to get involved, visit the GitHub repository. Make sure to also join the #ebpf-website Slack channel to get in touch with the team working on the documentation. Share what you have learned by improving the documentation or write additional tutorials.</p>',
+          links: [
+            {
+              text: 'License',
+              to: 'https://creativecommons.org/licenses/by/4.0/',
+              target: '_blank',
+            },
+            {
+              text: 'GitHub repository',
+              to: 'https://github.com/ebpf-io/ebpf.io-website',
+              target: '_blank',
+            },
+            {
+              text: 'Slack',
+              to: 'https://ebpf.io/slack',
+            },
+          ],
+        },
+      ],
+    },
+  },
```

Comment:
```suggestion
  pt: {
    hero: {
      title: 'Como Contribuir com o eBPF',
      description:
        'O eBPF consiste em muitas comunidades, incluindo o _runtime_ do eBPF no kernel Linux, vários SDKs e bibliotecas de desenvolvimento, uma série de grandes projetos usando eBPF e um amplo conjunto de projetos e ferramentas menores. Existem, portanto, muitas maneiras de contribuir com a comunidade eBPF e de se envolver. A lista de ideias a seguir pode servir como ponto de partida sobre como você pode se envolver e contribuir com a comunidade.',
      card: {
        title: 'Contribua com o runtime do eBPF no kernel Linux',
        description:
          'A comunidade do kernel Linux mantém árvores Git separadas para o subsistema eBPF para gerenciar todas as contribuições dos desenvolvedores. As árvores são mantidas por:',
        secondDescription:
          'Trabalhar no tempo de execução do eBPF pode ser incrivelmente gratificante, pois constrói a base para a gravação de software que roda em milhões de sistemas e dispositivos. O trabalho inclui uma ampla gama de tópicos, desde adição de suporte a ganchos adicionais, adição de novos tipos de programas, melhorias no verificador eBPF, adição de suporte JIT para arquiteturas de CPU adicionais, extensão do conjunto de auxiliares BPF, adição de novos casos de teste à infraestrutura de teste do BPF, ou melhorias no bpftool e libbpf.',
        authors: [
          {
            name: 'Alexei Starovoitov',
            company: 'Facebook',
          },
          {
            name: 'Daniel Borkmann',
            company: 'Cilium/Isovalent',
          },
        ],
        links: [
          {
            text: 'Git trees',
            to: 'https://git.kernel.org/?q=BPF+Group',
          },
          {
            text: 'Linux kernel eBPF runtime',
            to: '/infrastructure/#linux-kernel',
          },
        ],
      },
    },
    howToContribute: {
      items: [
        {
          icon: 'file',
          title: 'Contribua para um projeto eBPF',
          description:
            '<p>A lista de projetos baseados em eBPF é longa e crescente. Será simples encontrar um projeto que desperte interesse. Confira o panorama do eBPF para ter uma visão geral dos projetos baseados em eBPF.</p><p>Muitos dos projetos baseados em eBPF listados mantêm uma lista de boas tarefas rotuladas como o primeiro problema, com escopo que não requer amplo conhecimento específico do projeto e oferece uma grande oportunidade de colocar a mão na massa rapidamente.</p>',
          links: [
            {
              text: 'eBPF landscape',
              to: '/applications',
            },
          ],
        },
        {
          icon: 'gear',
          title: 'Contribua com as ferramentas de desenvolvimento do eBPF',
          description:
            '<p>Embora o kernel Linux forneça bibliotecas e APIs mínimas para usar o eBPF diretamente, a maioria dos projetos utiliza uma cadeia de ferramentas de desenvolvimento de eBPF para simplificar o uso do eBPF e compartilhar código comum entre outros projetos.</p><p>Confira as cadeias de ferramentas de desenvolvimento eBPF para obter uma lista atualizada frequentemente de projetos neste espaço. Você encontrará informações sobre como se envolver diretamente no site de cada projeto.</p>',
          links: [
            {
              text: 'eBPF development toolchains',
              to: '/what-is-ebpf/#development-toolchains',
            },
          ],
        },
        {
          icon: 'pen',
          title: 'Contribua com ebpf.io',
          description:
            '<p>Você tem talento para escrever? Você é bom com diagramas?O conteúdo do site ebpf.io está licenciado sob uma licença _Creative Commons Attribution 4.0 International_. Se você quiser se envolver, visite o repositório no GitHub. Não deixe de entrar também no canal #ebpf-website no Slack para entrar em contato com a equipe que está trabalhando na documentação. Compartilhe o que você aprendeu melhorando a documentação ou escrevendo tutoriais adicionais.</p>',
          links: [
            {
              text: 'License',
              to: 'https://creativecommons.org/licenses/by/4.0/',
              target: '_blank',
            },
            {
              text: 'GitHub repository',
              to: 'https://github.com/ebpf-io/ebpf.io-website',
              target: '_blank',
            },
            {
              text: 'Slack',
              to: 'https://ebpf.io/slack',
            },
          ],
        },
      ],
    },
  },
```

Reply:
@prosazhin @xmulligan I made the suggestion with the translation into Brazilian Portuguese for the contribute page.

Reply:
Great, @Hudsoncoutinho what do you think?